Compilation error because of setSystemPropertyOnServer

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

testsuite

### Describe the bug

Compilation fails on main. I suppose it was again several modifications at the same time in different PRs:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) on project integration-arquillian-tests-base: Compilation failure: Compilation failure: 
[ERROR] /home/rmartinc/keycloak/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/LightWeightAccessTokenTest.java:[996,37] cannot find symbol
[ERROR]   symbol:   method setSystemPropertyOnServer(java.lang.String,java.lang.String)
[ERROR]   location: interface org.keycloak.testsuite.client.resources.TestingResource
[ERROR] /home/rmartinc/keycloak/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/LightWeightAccessTokenTest.java:[997,37] cannot find symbol
[ERROR]   symbol:   method reinitializeProviderFactoryWithSystemPropertiesScope(java.lang.String,java.lang.String,java.lang.String)
[ERROR]   location: interface org.keycloak.testsuite.client.resources.TestingResource
[ERROR] -> [Help 1]
```

### Version

999.0.0-SNAPSHOT

### Regression

- [ ] The issue is a regression

### Expected behavior

Compilation should work.

### Actual behavior

Compilation errors.

### How to Reproduce?

```
mvn clean install -DskipTests
```

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/LightWeightAccessTokenTest.java
+++ b/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/LightWeightAccessTokenTest.java
@@ -993,8 +993,8 @@ public class LightWeightAccessTokenTest extends AbstractClientPoliciesTest {
     }
 
     private void allowUserinfoWithLightweightAccessToken(boolean allow) {
-        getTestingClient().testing().setSystemPropertyOnServer("oidc.allow-userinfo-with-lightweight-access-token", String.valueOf(allow));
-        getTestingClient().testing().reinitializeProviderFactoryWithSystemPropertiesScope(org.keycloak.protocol.LoginProtocol.class.getName(), OIDCLoginProtocol.LOGIN_PROTOCOL, "oidc.");
+        runOnServerMaster.run(RunHelpers.setSystemPropertyOnServer("oidc.allow-userinfo-with-lightweight-access-token", String.valueOf(allow)));
+        runOnServerMaster.run(RunHelpers.reinitializeProviderFactoryWithSystemPropertiesScope(org.keycloak.protocol.LoginProtocol.class.getName(), OIDCLoginProtocol.LOGIN_PROTOCOL, "oidc."));
     }
 
     @Test

